### DISCUSSION TITLE
Get rid of pendulum dependency? It's buggy and more or less unmaintained

### DISCUSSION THREAD

**Discussion by jonaslb** [PrefectHQ/prefect#9969] [Ideas]
I recently ran into a problem running prefect in a CI/CD pipeline due to a Pendulum bug (which prefect relies on). I realised that Pendulum has seen no updates in the last three years or so and it's looking for someone to do maintenance (ie author doesn't really have time for it).

Since Python's builtin `datetime` module has matured a lot in the last releases, and Pendulum's biggest selling point was sane timezone handling which `datetime` now also has, maybe Prefect should just get rid of it? It would have saved me a pain today at least, if this unnecessary dependency had been kept out.

I did [report the bug to pendulum](https://github.com/sdispater/pendulum/issues/715) but I wouldn't hold my breath to see it fixed. Basically, just use the stdlib `datetime` instead?

**Comment by Expan75** [comment]
Hit this as well: @jonaslb did you manage to solve the issue?

**Reply by jonaslb** [reply]
Pendulum claims to have fixed the issue, so I don't think you're seeing the same. I suggest you open an issue yourself and describe what you've run into.

**Comment by aditbisa** [comment]
Using poetry (which has same author as pendulum), the pendulum v3 require rust compiler to build the wheels.
I do not want to bloat my CI/CD base image more than what I did for poetry by adding build-essentials.

So. please remove pendulum out of prefect.

Errors:
```
15.75
15.75   Running `maturin pep517 build-wheel -i /tmp/tmp79gqgoro/.venv/bin/python --compatibility off`
15.75   💥 maturin failed
15.75     Caused by: Cargo metadata failed. Do you have cargo in your PATH?
15.75     Caused by: No such file or directory (os error 2)
15.75   Error: command ['maturin', 'pep517', 'build-wheel', '-i', '/tmp/tmp79gqgoro/.venv/bin/python', '--compatibility', 'off'] returned non-zero exit status 1
```
